eskipfile/remote.go

```diff
@@ -172,13 +181,24 @@ func (client *remoteEskipFile) DownloadRemoteFile() error {
 }
 
 func (client *remoteEskipFile) getRemoteData() (io.ReadCloser, error) {
+
+	client.isCached = false
+
 	resp, err := client.http.Get(client.remotePath)
 	if err != nil {
 		return nil, err
 	}
+
 	if resp.StatusCode != 200 {
+		if resp.StatusCode == 304 {
+			client.isCached = true
```

Comment:
another field another data race ;)
I think it's better to return a pre-defined error like:
```
ErrContentNotChanged = errors.New("content in cache did not change")
```
and then return this and check this where you need.

Reply:
you mean error instead of field right?